src/pages/contributors/jesse-glick.adoc

```diff
@@ -0,0 +1,107 @@
+A lot of time is spent on these challenges, along with making sure tests keep running and making sure it's possible to do routine updates. 
+
+== Why choose Jenkins over other projects?
+
+There weren't a lot of options at the time, with Cruise Control being the only other alternative.
+It worked and was very extensible.
+Jenkins either did what we needed it to or we could add plugins that would perform the necessary functions.
+After becoming familiar with the system, extending Jenkins further was fairly straightforward.
+For instance, we wanted to generate a historical diff for quick reference that anyone could use.
+This wasn't possible inherently with Jenkins, but we were able to create a plugin that completed this particular task.
+Jenkins' versatility has only expanded since then, proving to be one of its greatest strengths.
+
+== What problems have you solved in Jenkins?
+
+I'm always looking for opportunities to simplify some parts of the architecture where possible.
+One project I worked on was the re-design of the Jenkins CLI.
+At the time, it was very troublesome from a security perspective and we would have to issue emergency advisories for something like the controller sending back something malicious.
+The project goal was to rip out all of the problematic functionality and turn it into a dump client.
+It could send a command and get back a stream of text, and the protocol was simplified so that it couldn't do anything more than these basic functions.
+However, this also meant tracking down every plugin that might interact with the CLI in this way and either updating code as needed or deprecating the function so that this would not persist.
+
+I also did some work on plugable storage changes, mainly the link:https://plugins.jenkins.io/artifact-manager-s3/[Artifact Manager on S3] plugin, to help externalize the storage of artifacts and statuses.
+The plugin ecosystem also forces you to work in particular ways, such as keeping track of what plugins might be affected by a change, or determining how a plugin might interact with various functions or other plugins. 
+It also pushes us to track down plugin maintainers for assistance with changes or updates, to ensure things are not broken by Jenkins core updates.
+
+== Is there an aspect of Jenkins that you're particularly interested in?
+
+A lot of my work has been done on Pipeline so I am partial to that.
+There have been several requests or bug reports from users that have also driven the work we do within Pipeline and Jenkins as a whole.
+These requests also help drive architectural changes, despite the challenges they can present.
+Such requests also help find and deal with technical debt.
+
+== What contributions have felt the most successful or impactful?
+
+The changes and advancements in Pipeline, including Multibranch Pipeline and Scripted Pipeline.
+
+// Left mostly blank for Jesse to provide additional information/response
```

Comment:
```suggestion
I was eager to file one of the first substantive link:https://github.com/jenkinsci/jep/tree/master/jep[JEPs]:
JEP-200, greatly reducing the surface area of Jenkins threatened by deserialization “gadgets”
(in the same spirit as the Jenkins CLI simplification, which predated the JEP system).
Another notable feature was JEP-222, allowing WebSocket to be used to connect agents to the controller.
This was inspired by the trickiness of setting up “level 4” network proxies,
a common hurdle for CloudBees CI customers especially in Kubernetes.

JEP-227 was a very tedious, yet long overdue, project to update to the (then-) current Spring Security release across several major versions.
Preventing almost all user-visible regressions required a lot of refactoring in Jenkins core and a lot of plugin updates.
It is great to see that the Jenkins project has gotten much more proactive in updating dependencies since those days.

At the infrastructure level, JEP-305 made it possible to prototype changes spanning multiple plugins without sacrificing CI coverage,
despite the lack of a “monorepo” for Jenkins development.
More recently, JEP-229 built on that system to make it possible to publish plugin releases automatically
without requiring maintainers to keep Artifactory credentials on their laptop,
a longstanding source of frustrating mistakes (and a security risk).
Both changes were inspired especially by work on Pipeline-related plugins,
which form a loose coalition without a fixed boundary:
I wanted it to be as simple as possible to develop, test, and publish changes rapidly.
```